### PR TITLE
feat(gmail): add localization support

### PR DESCRIPTION
**Pull request by EncryptedDev**
PreMiD/Presences#5216

### CODE REVIEW

**Comment by EncryptedDev** [comment]
@ImRodry

File: src/Presence/gmail.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  "gmail.categoryUpdates": {
+    "message": "Viewing the Updates category",
+    "description": "Displayed when viewing updates emails."
+  },
+  "gmail.categoryPromotions": {
+    "message": "Viewing the Promotions category",
+    "description": "Displayed when viewing promotions emails."
+  },
+  "gmail.categoryForum": {
+    "message": "Viewing the Forum category",
```
Comment (ImRodry) [inline src/Presence/gmail.json:19]:
```suggestion
    "message": "Viewing the Forums category",
```

File: src/Presence/gmail.json [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  "gmail.inLabel": {
+    "message": "Viewing label",
+    "description": "Displayed when a user is browsing emails in a certain label. The name of the label will appear below."
+  },
+  "gmail.generalSettings": {
+    "message": "Configuring General settings",
+    "description": "Displayed when the user is modifying their settings"
+  },
+  "gmail.labelSettings": {
+    "message": "Configuring Labels settings",
```
Comment (ImRodry) [inline src/Presence/gmail.json:31]:
```suggestion
    "message": "Configuring Label settings",
```

File: src/Presence/gmail.json [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  "gmail.inboxSettings": {
+    "message": "Configuring Inbox settings",
+    "description": "Displayed when the user is modifying their inbox settings"
+  },
+  "gmail.accountSetting": {
+    "message": "Configuring Account and Import settings",
+    "description": "Displayed when the user is modifying their account's settings"
+  },
+  "gmail.filterSettings": {
+    "message": "Configuring Filters and Blocked Addresses settings",
```
Comment (ImRodry) [inline src/Presence/gmail.json:43]:
```suggestion
    "message": "Configuring Filters and Blocked Addresses",
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by ImRodry — APPROVED** [review]

**Pull request merged**